.github/workflows/node-zxc-build-release-artifact.yaml

```diff
@@ -188,7 +192,16 @@ jobs:
         id: effective-version
         run: |
           EFF_VERSION="$(./gradlew showVersion --quiet | tr -d '[:space:]')"
+          PRERELEASE_SUFFIX="$(semver get prerel "${EFF_VERSION}")"
+          PRERELEASE="false"
+          [[ -n "${PRERELEASE_SUFFIX}" ]] && PRERELEASE="true"
+
+          MAJOR_VERSION="$(semver get major "${EFF_VERSION}")"
+          MINOR_VERSION="$(semver get minor "${EFF_VERSION}")"
+
           echo "number=${EFF_VERSION}" >>"${GITHUB_OUTPUT}"
+          echo "prefix=${MAJOR_VERSION}.${MINOR_VERSION" >>"${GITHUB_OUTPUT}"
```

Comment:
A bracket seems to be missing for MINOR_VERSION

Reply:
Wells thats a bummer, it means `develop` is horked as well. Lets merge the cherry pick as is and then I can chase it with a followup PR and cherry pick again.